File: frontend/gamesoldfrontprojkt/src/app/components/model/OrderDTO.ts

export class OrderDTO {
    userName: String;
    idBuyedGames: Array<String>;
    orderPrice: number;
    orderIsPayed: boolean;
    paymentMethod: String;

    constructor(userReceivedName: String, buyedGamesIdReceived: Array<String>, receivedOrderPrice: number, receivedOrderIsPayed: boolean, receivedPaymentMethod: String){
        this.userName = userReceivedName;
        this.idBuyedGames = buyedGamesIdReceived;
        this.orderPrice = receivedOrderPrice;
        this.orderIsPayed = receivedOrderIsPayed;
        this.paymentMethod = receivedPaymentMethod;
    }
}

File: src/main/java/gamesoldstoreprojkt/Model/OrderDTO.java
package gamesoldstoreprojkt.Model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class OrderDTO {
    private String userName;
    private String [] idBuyedGames;
    private String orderPrice;
    private boolean orderIsPayed;
    private String paymentMethod;
    
    
    public boolean getOrderIsPayed(){
        return this.orderIsPayed;
    }
}
